src/cocotb/share/lib/simulator/simulatormodule.cpp

```diff
@@ -992,6 +1171,14 @@ static PyMethodDef SimulatorMethods[] = {
                "--\n\n"
                "get_simulator_version() -> str\n"
                "Get the simulator's product version string.")},
+    {"clock_create", clock_create, METH_VARARGS,
+     PyDoc_STR("clock_create(signal, /)\n"
+               "--\n\n"
+               "clock_create(signal: cocotb.simulator.gpi_sim_hdl"
+               ") -> cocotb.simulator.GpiClock\n"
+               "Create a clock driver on a signal."
+               "\n"
```

Comment:
```suggestion
               "\n"
               "\n"
```
Fix invalid markup (https://cocotb--3983.org.readthedocs.build/en/3983/library_reference.html#cocotb.simulator.clock_create)